### ISSUE TITLE
Demographics of Cambridge riders: age, gender

### ISSUE THREAD

**Issue by itamarst** [itamarst/cambridge-bicycle-usage#2]
Data is available from (part of) 2020 and earlier

**Comment by itamarst** [comment]
Demographics over time might be interesting too (2016-2020)

**Comment by sckilcoyne** [comment]
this is from the BB data?

**Comment by itamarst** [comment]
Yep, up until early 2020 they report year of birth (maybe check for anomalies in case people just pick random number) and the three genders (0, 1, and 2).
